install/kubernetes/Makefile

```diff
@@ -89,16 +89,19 @@ update-versions:
 		branch="$$cilium_version";											\
 		hubble_version=$(HUBBLE_UI_VERSION);										\
 		pull_policy="IfNotPresent";											\
+		use_digest="false";											\
```

Comment:
This was 'true' in the original commit, but I've changed it to 'false' in this PR so that the helm charts disable digests by default.

As a result, generated helm charts will have them disabled; however, the experimental / quick installs will enable them by default using the result of `USE_DIGEST` at the top of this commit. I believe this is consistent with how the branch is currently operating, but with one extra improvement: When preparing the release commit for the next release (when there are no valid digests), rather than setting the image tags in such generated YAMLs (used in smoke tests) falsely to `cilium/cilium:latest@`, it will instead generate `cilium/cilium:latest`. After we fill in the digests post-release, the quick install will point to actual image shas for the new release.

Reply:
shouldn't point to `cilium/cilium:v1.9` instead?

Reply:
`cilium/cilium:v1.9.6` or similar, yeah. `latest` was a typo on my side, my intent was to emphasize the extra `@` which breaks the test.